static/js/popover_menus.js

```diff
@@ -56,6 +56,15 @@ function on_show_prep(instance) {
     popovers.hide_all_except_sidebars(instance);
 }
 
+function disable_tooltip_when_menu_open() {
```

Comment:
Let's call this `update_stream_settings_tooltip_appearance` or something like that.

Reply:
Thanks for responding @amanagr, I have done the required changes. Can you please review the PR again.